doc/quickstart/ncWMS_quickstart.rst

```diff
@@ -1,38 +1,41 @@
 :Author: Guy Griffiths (guy.griffiths@the-iea.org)
-:Version: osgeolive7.0
+:Reviewer: Felicity Brand (Google Season of Docs 2019)
+:Version: osgeolive12.0
 :License: Creative Commons Attribution-ShareAlike 3.0 Unported  (CC BY-SA 3.0)
 
 @LOGO_ncWMS@
 @OSGEO_KIND_ncWMS@
 
 
-********************************************************************************
+***********************
 @NAME_ncWMS@ Quickstart
-********************************************************************************
+***********************
 
 ncWMS is a Web Map Service which allows the fast visualisation of data from NetCDF files and other sources of environmental data.  This quickstart guide describes how to explore the sample data provided using the Godiva2 web client.  For configuration and adding other data sources to the server, please consult the documentation on the `ncWMS website <https://reading-escience-centre.gitbooks.io/ncwms-user-guide/content/>`_.
 
 .. contents:: Contents
+   :local:
 
-Start ncWMS
-===========
 
-* Select :menuselection:`Geospatial --> Web Services --> ncWMS --> Start ncWMS`
+Start ncWMS and understand the workspace
+========================================
 
-* After a few moments the application will start up and open a web browser at http://localhost:8080/ncWMS2/Godiva3.html
+Select :menuselection:`Geospatial --> Web Services --> ncWMS --> Start ncWMS`.
+
+After a few moments the application will start up and open a web browser at http://localhost:8080/ncWMS2/Godiva3.html
 
 .. image:: /images/projects/ncWMS/ncWMS-01-start_screen.png
     :scale: 55 %
 
-Basic usage
-===========
 
-Use the left-hand menu to choose a dataset to view.  Clicking a dataset (or the + icon to the left of it) will expand the dataset to show the available variables to plot.  Choose a variable by clicking on it.  The data should appear on the interactive map after a short delay (a progress bar may appear showing the progress of loading image tiles from the WMS server(s)).
+* Use the left-hand menu to choose a dataset to view.  
```

Comment:
It would be better for this document if we chose one scenario and stepped readers through it, rather than listing what you _can_ do. So we should tell them a dataset to choose, then tell them a variable to display and how to color and style the map. That would make this more of a Quickstart. I would be happy to help do this, but don't know enough about the application to write it on my own.